### DISCUSSION TITLE
Getting errors about multiple instances `graphql` after upgrading to 1.3.1

### DISCUSSION THREAD

**Discussion by oskarkivra** [vitest-dev/vitest#5375] [Q&A] [answered]
Hi,

I have been using version 0.34.6 (the latest version before 1.x) and have multiple tests where I create a graphql server, sends requests and validating the result. It has been working great. However, once I upgrading to 1.3.1 all tests are failing with the message:
```
Cannot use GraphQLObjectType "Query" from another module or realm.

 Ensure that there is only one instance of "graphql" in the node_modules
 directory. If different versions of "graphql" are the dependencies of other
 relied on modules, use "resolutions" to ensure only one version is installed.

 https://yarnpkg.com/en/docs/selective-version-resolutions

 Duplicate "graphql" modules cannot be used at the same time since different
 versions may have different capabilities and behavior. The data from one
 version used in the function from another could produce confusing and
 spurious results.
```

According to pnpm do I only have one version of `graphql` (`pnpm ls -r graphql` / `pnpm why graphql`), I have never the less tried to add `resolve.dedupe: ['graphql']` in my vite config without any luck.

I guess it has something todo with the new pool options? Or can it be something else? It works fine if I downgrade to 0.34.6 again. What flag can I use to get the same pool/thread behavior as in 0.34.6? Or is it the same by default?

I'm using node 20.11.1, pnpm 8.15.4 and this is my vite test config:
```ts
import { defineConfig } from 'vitest/config';

export default defineConfig({
  test: {
    globals: true,
  },
});
```

**Comment by AriPerkkio** [comment, marked as answer]
This also came up at https://github.com/vitest-dev/vitest/issues/4605. Have you tried solutions from there?

**Reply by oskarkivra** [reply]
Yes, thanks. It fix my issue.
